tools/pipeline-owners-extractor/PipelineOwnersExtractor.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
What is this file used by?

Reply:
ReSharper; I added a dictionary entry for `Codeowners` so it won't complain.